ibm_catalog.json

```
@@ -0,0 +1,279 @@
+{
+    "products": [
+        {
+            "name": "deploy-arch-ibm-is-private-path-external-connectivity",
+            "label": "Cloud essentials for private-path external connectivity",
+            "product_kind": "solution",
+            "tags": [
+                "ibm_created",
+                "target_terraform",
+                "terraform",
+                "solution",
+                "network"
+            ],
+            "keywords": [
+                "vpc",
+                "private path",
+                "private",
+                "nlb",
+                "IaC",
+                "infrastructure as code",
+                "terraform",
+                "solution"
+            ],
+            "short_description": "Creates and configures a VPC Private Path service for external connectivity.",
+            "long_description": "This solution is used to provision and configure a VPC Private Path service for external connectivity.",
+            "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-vpc-private-path-external-connectivity/blob/main/solutions/fully-configurable/README.md",
+            "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-vpc-private-path-external-connectivity/main/images/private_path.svg",
+            "provider_name": "IBM",
+            "features": [
+                {
+                    "title": "Creates a Application load balancer.",
+                    "description": "Sets up an IBM Application load balancer with a backend pool which will connect to the external service."
+                },
+                {
+                    "title": "Creates a Private Path service.",
+                    "description": "Creates an IBM VPC Private Path service."
+                },
+                {
+                    "title": "Sets up a Private Path Network load balancer.",
+                    "description": "Sets up an IBM Private Path Network load balancer with a backend pool which will connect to the VPE Gateway."
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "Fully Configurable",
```

just spotted this while creating the tile - the label should be: 
```suggestion
                    "label": "Fully configurable",
```